maven-archetype-plugin/src/main/java/org/apache/maven/archetype/ui/ArchetypeConfiguration.java

```diff
@@ -19,12 +19,15 @@
  * under the License.
  */
 
+import org.apache.maven.archetype.common.Constants;
+import org.codehaus.plexus.util.StringUtils;
```

Comment:
okay, I'll put it back to the wrong place (it's the only file where the org.* imports are after the java.* imports)